lambdas/shared/tests/test_decorator.py

```diff
@@ -0,0 +1,81 @@
+"""
+Decorator tests
+"""
+
+from unittest import TestCase
+
+from t4_lambda_shared.decorator import api, validate
+
+class TestDecorator(TestCase):
+    def test_api_basic(self):
+        @api(cors_origins=['https://example.com'])
```

Comment:
it would be nice to have a unit test that tosses a few broad errors/exceptions

Reply:
I'm already testing a `TypeError`, which I think is pretty broad?

Reply:
sure; but that's in test_index right now. belongs in test_decorator, right? e.g. if we lift this code to another project there will be no unit test for errors in test_decorator.

Reply:
Ohh, you're right. I'll move the test here - it's not really testing the preview endpoint